lib/sms.php

```diff
@@ -43,10 +43,18 @@ function send_sms( $to, $message ) {
 			}
 
 			$body = array(
-				'From' => SMS_FROM_NUMBER,
 				'To'   => $to_number,
 				'Body' => $message_split,
 			);
+			/**
+			 * If defined, we want to use the MessagingServiceSid to leverage all the automatic logic of the messaging service to route the SMS from the right "From" number.
+			 * For example in some countries, the "From" number should be a local number to the recipient or an alphanumeric sender ID.
+			 */
+			if ( defined( 'TWILIO_MESSAGING_SERVICE_SID' ) ) {
```

Comment:
@rebeccahum @andrea-sdl can we actually prefix this with `VIP_`?

I know there are other constants in this file that aren't prefixed, but it would be a good idea to do so for any new and safe to change constants.

Reply:
@mjangda added in https://github.com/Automattic/vip-go-mu-plugins/pull/5737